v2/azure-arm.yaml

```diff
@@ -880,6 +885,15 @@ objectModelConfiguration:
       Service_ApiVersionSet:
         $exportAs: ApiVersionSet
         $supportedFrom: v2.4.0
+      Service_AuthorizationProvider:
+        $exportAs: AuthorizationProvider
+        $supportedFrom: v2.4.0
+      Service_AuthorizationProviders_Authorization:
+        $exportAs: Authorization
+        $supportedFrom: v2.4.0
+      Service_AuthorizationProviders_Authorizations_AccessPolicy:
+        $exportAs: AccessPolicy
+        $supportedFrom: v2.4.0
```

Comment:
This should be 2.6.0, no?

Reply:
there is a commented section for 2.5.0, so I assumed its not released yet for apim

Reply:
Also for that we would need to define apim resource for all of 2.5.0, then we can do it to for 2.6.0
line 928
"# TODO: New APIM version
    # 2023-05-01-preview:
    #   Service:
    #     $export: true
    #     $supportedFrom: v2.5.0
    #   CertificateConfiguration:
    #     CertificatePassword:
    #       $isSecret: true
    #   HostnameConfiguration:
    #     CertificatePassword:
    #       $isSecret: true
    #     IdentityClientId:
    #       $importConfigMapMode: optional"